### PR TITLE
Adding link in Icons section

### LINKED ISSUE #1824: [Add]  Adding a link in, category- `frontend`, sub-category - `Icons`
### Site URL

https://evil-icons.io/

### Description of the above provided site

Simple and clean SVG icon pack with the code to support Rails, Sprockets, Node.js, Gulp, Grunt and CDN.

### Category

Frontend

### Sub Category

Icons

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by Prakhar-Shankar**
## Fixes Issue

Closes #1824 

## Changes proposed

Added a link in Icons section

## Note to reviewers

Would love to add more contributions.

### CODE REVIEW

**Comment by CBID2** [comment]
Hi @Prakhar-Shankar! :) Welcome to LinksHub! :) We don't accept spam pull requests, so I'm putting yours in draft mode so I can review your issue. In the future, please wait until your issue has been approved and assigned to you before you submit a pull request. Refer to our [Contributing Guide](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md) to learn how to effectively contribute to the project. Happy Contributing! 😊

File: database/frontend/icons.json [hunk 1]
```diff
@@ -222,5 +222,12 @@
     "url": "https://iconoir.com/",
     "category": "fontend",
     "subcategory": "icons"
+  },
+  {
+    "name": "Evil Icons",
+    "description": "Simple and clean SVG icon pack with the code to support Rails, Sprockets, Node.js, Gulp, Grunt and CDN.",
```
Comment (CBID2) [inline database/frontend/icons.json:228]: [proofreading] There's a missing comma after "Grunt" 
```suggestion
    "description": "Simple and clean SVG icon pack with the code to support Rails, Sprockets, Node.js, Gulp, Grunt, and CDN.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Prakhar-Shankar! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or spell check before making a pull request. Other than that, you're off to a great start! :)

**Comment by Prakhar-Shankar** [comment]
> Hi @Prakhar-Shankar! :) I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or spell check before making a pull request. Other than that, you're off to a great start! :)

Hi, thank you for the suggestion. I will take care of this in the future.

**Review by rupali-codes — APPROVED** [review]

**Comment by rupali-codes** [comment]
> Reference in ne

Its alright :)

**Pull request merged**